src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -159,9 +159,9 @@ public void ProcessStart_UseShellExecute_OnUnix_SuccessWhenProgramInstalled(bool
         [PlatformSpecific(~TestPlatforms.OSX)] // On OSX, ProcessName returns the script interpreter.
         public void ProcessNameMatchesScriptName()
         {
-            string scriptName = GetTestFileName();
+            const string scriptName = nameof(ProcessNameMatchesScriptName);
```

Comment:
Unless the underlying filesystem imposes additional restrictions, Linux filenames can contain any bytes except `\0` and `/`.
.NET assumes the bytes are UTF-8.
So, I think the filename is not an issue. If it were, we should make sure `GetTestFileName` returns a valid filename on Linux instead of doing something specific in this test.

Reply:
I agree I do'nt see a problem with GetTestFileName(), and if it has a problem, it will be affecting other tests. Either way no change is needed here.